README.md

```diff
@@ -54,4 +58,33 @@ To download MATLAB go [here](https://www.mathworks.com/downloads/).
 ### Python
 We will be using Python 3.6 with the Anaconda distribution. This is a large package that includes many of the base functions and IDE's that are very common to Python.
 
-To download the Anaconda distribution, go [here](https://www.anaconda.com/products/individual) and choose your installer near the bottom. Anaconda comes with a few possible IDE's, we prefer VScode but you can use whichever to edit the scripts and run them.
\ No newline at end of file
+To download the Anaconda distribution, go [here](https://www.anaconda.com/products/individual) and choose your installer near the bottom. Anaconda comes with a few possible IDE's, we prefer VScode but you can use whichever to edit the scripts and run them.
+
+## 4. Creating the Git Repository for your group.
+This section should cover how the group leader will download the repository, create their own version of it, and then invite their group members (collaborators) to the project. Only one person per group needs to do this.
+
+### a. Download the repo in .zip format.
+First, download the current repo in a .zip format. Go the the primary repository [here](https://github.com/alcantarar/ASB_Tutorial), then near the top right of the window click the green button with "Code" on it. Then select **Download ZIP**. Extract this zip file to somewhere on your computer.
+
+![](media/zip_download.png)
+
+### b. Create a repository on your Github account.
+To create a new repository in your Github account, click the button in the very top left of this window (green circle), then near there click **New** (red circle).
+
+![](media/new_repo.png)
+
+Title this however you prefer, and make sure the repository is public. 
```

Comment:
Make name consistent so that we can just directly reference it in our instructions. `ASB_Tutorial_Example` or something.